WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/BlockEditorPage.java

```diff
@@ -74,10 +75,25 @@ public BlockEditorPage addPostSettings(String categoryName, String tagName) {
         return this;
     }
 
-    public void openPostSetting() {
+    /**
+     * Taps the three vertical dots menu button located at the editor screen top right.
+     * Since the tap does not always succeed on FTL, one retry is used
+     */
+    public void openPostKebabMenu() {
         waitForElementToBeDisplayed(R.id.toolbar_main);
+
+        // First attempt to tap the kebab menu (three dots)
         openActionBarOverflowOrOptionsMenu(ApplicationProvider.getApplicationContext());
-        clickOn(onView(withText(R.string.post_settings)));
+
+        // Check if the attempt succeeded, retry once if not
+        if (!waitForElementToBeDisplayedWithoutFailure(postSettingButton)) {
+            openActionBarOverflowOrOptionsMenu(ApplicationProvider.getApplicationContext());
+        }
```

Comment:
👋 @pachlava! 
Have you considered failing the test in case the menu is not displayed after the second tap?

Reply:
While my first thought was "the test will fail anyway right on the next line", this only applies to the current state of tests, because this function is used in only one place. If it will be used in other cases, your suggestion makes a lot of sense, and I committed it with XXX. Thanks, @tiagomar 👋